guides/cookbook.md

```diff
@@ -1102,6 +1425,53 @@ while (type = reader.next()) {
 ```
 </div>
 
+<div class="tabpane Python" markdown="1">
+#### Using simpleion
+`load/loads` always fully materializes the entire stream. Hence, a sparse read that leverages the
+efficiencies of binary Ion is not possible via this API. With that in mind, it is possible to
+sparsely access the fully-materialized values.
+
+```python
+data = ... # The binary Ion equivalent of the above data.
```

Comment:
copy/paste is powerful, let's help 'em out:
```
from amazon.ion import simpleion
# The binary Ion equivalent of the above data:
data = b'\xe0\x01\x00\xea' \
    b'\xee\xa5\x81\x83\xde\xa1\x87\xbe\x9e\x83\x66\x6f\x6f\x88' \
    b'\x71\x75\x61\x6e\x74\x69\x74\x79\x83\x62\x61\x72\x82\x69' \
    b'\x64\x83\x62\x61\x7a\x85\x69\x74\x65\x6d\x73\xe6\x81\x8a' \
    b'\xd3\x8b\x21\x01\xe9\x81\x8c\xd6\x84\x81\x78\x8d\x21\x07' \
    b'\xee\x95\x81\x8e\xde\x91\x8f\xbe\x8e\x86\x74\x68\x69\x6e' \
    b'\x67\x31\x86\x74\x68\x69\x6e\x67\x32\xe6\x81\x8a\xd3\x8b' \
    b'\x21\x13\xe9\x81\x8c\xd6\x84\x81\x79\x8d\x21\x08'
```

Reply:
Yuck. Done.